Validation/HGCalValidation/scripts/makeHGCalValidationPlots.py

```diff
@@ -5,28 +5,29 @@
 import argparse
 from time import time
 
+from RecoHGCal.TICL.iterativeTICL_cff import ticlIterLabels, ticlIterLabelsMerge
+
 from Validation.RecoTrack.plotting.validation import SeparateValidation, SimpleValidation, SimpleSample
 import Validation.HGCalValidation.hgcalPlots as hgcalPlots
 import Validation.RecoTrack.plotting.plotting as plotting
 
-simClustersIters = ["ClusterLevel","ticlTrackstersTrkEM","ticlTrackstersEM","ticlTrackstersTrk","ticlTrackstersHAD","ticlSimTracksters"]
+simClustersIters = ["ClusterLevel","ticlSimTracksters"]
+simClustersIters.extend('ticlTracksters'+iteration for iteration in ticlIterLabels)
 
-trackstersIters = ["ticlMultiClustersFromTrackstersMerge", "ticlMultiClustersFromTrackstersMIP",
-                   "ticlMultiClustersFromTrackstersTrk","ticlMultiClustersFromTrackstersTrkEM",
-                   "ticlMultiClustersFromTrackstersEM", "ticlMultiClustersFromTrackstersHAD",
-                   "ticlMultiClustersFromSimTracksters"]
+trackstersIters = ['ticlTracksters'+iteration for iteration in ticlIterLabelsMerge]
+trackstersIters.extend(["ticlSimTracksters"])
 
 simClustersGeneralLabel = 'simClusters'
 layerClustersGeneralLabel = 'hgcalLayerClusters'
-multiclustersGeneralLabel = 'hgcalMultiClusters'
-trackstersGeneralLabel = 'allTiclMultiClusters'
+trackstersGeneralLabel = 'hgcalTracksters'
+allTrackstersLabel = 'allTiclTracksters'
```

Comment:
can the names and the collection options `simClusters`, `hgcalTracksters`, `allTiclTracksters` made more clear and explicit. 
It's not clear at all what collection should people use to validate tracksters..

Reply:
Yes, I will take care of this in the next PR.